app/lib/dartdoc/dartdoc_page.dart

```diff
@@ -0,0 +1,419 @@
+        d.script(type: 'text/javascript', src: staticUrls.gtmJs),
+        d.meta(charset: 'utf-8'),
+        d.meta(httpEquiv: 'X-UA-Compatible', content: 'IE=edge'),
+        d.meta(
+            name: 'viewport',
+            content:
+                'width=device-width, height=device-height, initial-scale=1, user-scalable=no'),
+        d.meta(name: 'generator', content: 'made with love by dartdoc'),
+        d.meta(name: 'description', content: description),
+        d.element('title', text: title),
+        // HACK: Inject a customized canonical url
+        d.link(rel: 'canonical', href: options.canonicalUrl),
+        // HACK: Inject alternate link, if not is latest stable version
+        if (!options.isLatestStable)
+          d.meta(rel: 'alternate', href: options.latestStableDocumentationUrl),
+        d.link(rel: 'preconnect', href: 'https://fonts.gstatic.com'),
+        // HACK: This is not part of dartdoc
+        d.link(
+            rel: 'stylesheet',
+            type: 'text/css',
+            href: staticUrls.githubMarkdownCss),
+        // TODO: Consider using same github.css we use on pub.dev
+        d.link(
+            rel: 'stylesheet',
+            type: 'text/css',
+            href: staticUrls.dartdocGithubCss),
+        d.link(
+          rel: 'stylesheet',
+          href:
+              'https://fonts.googleapis.com/css2?family=Roboto+Mono:ital,wght@0,300;0,400;0,500;0,700;1,400&amp;display=swap',
+        ),
+        d.link(
+          rel: 'stylesheet',
+          href:
+              'https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@48,400,0,0',
+        ),
+
+        d.link(rel: 'stylesheet', href: staticUrls.dartdocStylesCss),
+        d.link(rel: 'icon', href: staticUrls.smallDartFavicon),
+      ]);
+
+  // HACK: Breadcrumbs are different from what dartdoc produces!
+  List<Breadcrumb> _breadcrumbs(DartDocPageOptions options) => [
+        // Prepend a '<name> package' breadcrumb leading back to pub.dev
+        Breadcrumb(
+          title: '${options.package} package',
+          href: options.pubPackagePageUrl,
+        ),
+        // Change the title of the first breadcrumb to be 'documentation'
+        if (breadcrumbs.firstOrNull != null)
+          Breadcrumb(
+            title: 'documentation',
+            href: breadcrumbs.first.href,
+          ),
+        ...breadcrumbs.skip(1),
+      ];
+
+  d.Node _renderHeader(DartDocPageOptions options) =>
+      d.element('header', id: 'title', children: [
+        d.span(
+          id: 'sidenav-left-toggle',
+          classes: ['material-symbols-outlined'],
+          attributes: {'role': 'button', 'tabindex': '0'},
+          text: 'menu',
+        ),
+        d.a(
+          href: '/',
+          classes: ['hidden-xs'],
+          child: d.img(
+            // TODO: Move this into a class
+            attributes: {'style': 'height: 30px; margin-right: 1em;'},
```

Comment:
I hope that we have tests that prevent this, but if we don't have it yet, we should ;)

Reply:
afaik, we are serving this right now :see_no_evil: